CONTRIBUTING.md

```diff
@@ -87,14 +87,73 @@ We use `mkdocs` to build the documentation and a plugin called `keras-autodoc` t
     python auto_doc.py
     ```
 
+##### Option 1: Build only current version of docs
+
 4. Either build the docs, or serve them dynamically:
 
+    Note: Links and pictures might not resolve properly later on when checking with this build.
```

Comment:
I would maybe put the reason why they would not: `as they are built to work with the versioning enabled` or something like that.

Reply:
added